src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -192,7 +192,14 @@ export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
   const [anchorEl, setAnchorEl] = useState(null);
   const [selectedCQLLibrary, setSelectedCqlLibrary] =
     useState<CqlLibrary>(null);
-  const isOwner = checkUserCanEdit(selectedCQLLibrary?.librarySet?.owner, []);
+  const canEdit = checkUserCanEdit(
+    selectedCQLLibrary?.librarySet?.owner,
+    selectedCQLLibrary?.librarySet?.acls
+  );
+  const isOwner = checkUserCanDelete(
```

Comment:
I think variable name `canDelete` would describe the result of this function more appropriately

Reply:
thank you I just changed it